api/candlepin-api-spec.yaml

```diff
@@ -1215,6 +1215,64 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /crl:
+    get:
+      tags:
+        - Certificate Revocation List
+      description: Retrieves the Certificate Revocation List
+      operationId: getCurrentCrl
+      security: []
+      x-java-response:
+        type: javax.ws.rs.core.Response
+        isContainer: false
+      responses:
+        200:
+          description: Certificate Revocation List successfully retrieved
```

Comment:
Let's avoid capitalization of the display message.

Reply:
Done.